aoc/src/main/scala/aoc2015/Day01_2015.scala

```diff
@@ -0,0 +1,39 @@
+object Day01_2015 extends App:
+  import scala.io.Source
+  import scala.collection.mutable.ListBuffer
+
+  val lines = Source.fromFile("aoc\\src\\resources\\aoc2015\\Day01_2015").getLines.mkString
```

Comment:
scala and java accept both unix and windows file naming - we prefer forward slashes as path separators - ie. "aoc/src.resources/aoc2015/Day01_2015"